Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -19,12 +19,11 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher
 {
     public class SearchSkimmer : Skimmer<AnalyzeContext>
     {
-        public const string AssetFingerprint = "AssetFingerprint/v1";
-        public const string ValidationFingerprint = "ValidationFingerprint/v1";
-        public const string ValidationFingerprintHash = "ValidationFingerprintHash/v1";
+        public const string UniqueSecretHashCurrent = "UniqueSecretHash256/current";
 
-        public const string AssetFingerprintV2 = "AssetFingerprint/v2";
-        public const string ValidationFingerprintV2 = "ValidationFingerprint/v2";
+        public const string AssetFingerprintCurrent = "AssetFingerprint/current";
```

Comment:
@michaelcfanning I am now a little uncertain about this versioning using `current` and `previous`.